frontend/src/pages/agreements/approve/ApproveAgreement.js

```diff
@@ -160,38 +176,69 @@ const ApproveAgreement = () => {
                     secondaryButtonText={modalProps.secondaryButtonText}
                 />
             )}
+            <section className="border-error border-dashed padding-1 margin-bottom-2">
```

Comment:
@stevtek will need some help with the workflow action. This section can be removed once we have that from the backend.